extras/help_generator/writers.py

```diff
@@ -146,24 +146,26 @@ def write_help_html(doc_dic, helpdir, fname, sli_command_list, keywords):
 
 def write_helpindex(helpdir):
     """
-    Returns index.html and index.hlp
-    --------------------------------
+    Writes helpindex.html and helpindex.hlp
+    ---------------------------------------
 
     """
-    filelist = glob.glob(helpdir + "/*/*.hlp")
+    print(helpdir)
+    filelist = glob.glob(os.path.join(helpdir, '*', '*.hlp'))
     html_list = []
     hlp_list = []
+    print(filelist[:10])
 
```

Comment:
Are these `print`s accidentally left from debugging? If not, I find them a bit unmotivated in the middle of all other possible output of the `generate_help` target.